src/ontology/config/icd10cm_term_exclusions.txt

```diff
@@ -0,0 +1,171 @@
+ICD10CM:B95-B97
```

Comment:
These seem very few: Given that src/ontology/config/icd10cm_exclusions.tsv has 115 rows, and most of them exclude all children, I expected this list to bein the 1000+ range, perhaps even 10K!

Reply:
@matentzn Hmm... Potentially what I could do is, rather than trying to analyze why this is low, I could use OAK instead of SPARQL and see if that changes anything.

FYI, the sparql code that generates this is:
`{{property_path_qualifier}}` below expands to `+` by default, but can also expand to `*` depending on an additional flag I can pass to the Python script. However, I'm not sure if this has anything to do with the issue.
```sparql
select ?term_id ?child_id ?child_label where {
  VALUES ?term_id { {{values}} }
  ?child_id rdfs:subClassOf{{property_path_qualifier}} ?term_id ;
    skos:prefLabel ?child_label .
}
```

Reply:
This is really weird and annoying. I responded to like 5 of your comments and my responses disappeared from GitHub.

I think you are meant to refer to `icd10cm_exclusion_reasons.robot.template.tsv`, which has ~173 rows. 

I wonder if there's an issue w/ the query? `property_path_qualifier` is `+` by default, but can also be `*` depending on a flag. However I'm not sure if that is the issue.
```sparql
select ?term_id ?child_id where {
  VALUES ?term_id { {{values}} }
  ?child_id rdfs:subClassOf{{property_path_qualifier}} ?term_id .
}
```

My plan is to migrate to OAK and see if that changes anything.

Reply:
ok now my comment showed up again...

Reply:
Just always use * instead of +. Does this change the number of classes?

Reply:
I haven't checked but I will make the change now.